src/models/transfers.js

```diff
@@ -481,11 +473,27 @@ function * insertTransfers (externalTransfers) {
     converters.convertToInternalTransfer))
 }
 
+function * resendNotification (transferId) {
```

Comment:
Yeah, that makes sense. I think we can expect clients to be robust enough to handle duplicate notifications, should they occur.